### ISSUE TITLE
Error in GetTextureFromCel

### ISSUE THREAD

**Issue by advra** [martinhodler/unity-aseprite-importer#16]
**Describe the bug**
When importing a file I get an error within GetTextureFromCel() of AseFile.cs. Here is the traceback in Unity:
```IndexOutOfRangeException: Array index is out of range.
Aseprite.AseFile.GetTextureFromCel (Aseprite.Chunks.CelChunk cel) (at Assets/unity-aseprite-importer/Editor/Aseprite/AseFile.cs:238)
Aseprite.AseFile.GetFrame (Int32 index) (at Assets/unity-aseprite-importer/Editor/Aseprite/AseFile.cs:174)
Aseprite.AseFile.GetFrames () (at Assets/unity-aseprite-importer/Editor/Aseprite/AseFile.cs:53)
AsepriteImporter.AseFileImporter.OnImportAsset (UnityEditor.Experimental.AssetImporters.AssetImportContext ctx) (at Assets/unity-aseprite-importer/Editor/AseFileImporter.cs:38)
UnityEditor.Experimental.AssetImporters.ScriptedImporter.GenerateAssetData (UnityEditor.Experimental.AssetImporters.AssetImportContext ctx) (at C:/buildslave/unity/build/Editor/Mono/AssetPipeline/ScriptedImporter.cs:19)
```

The culprit seems to be this line:
`
colors[index] = cel.RawPixelData[celDataIndex].GetColor();
`

Here is a snippet of some print statements checking index and celDataIndex. 
![image](https://user-images.githubusercontent.com/1753803/68532205-4659d680-02cf-11ea-9842-28f0c046e97b.png)

Im not entirely sure what the functions intended purpose is so it is difficult to determine what it does. "render as large as the viewport". Is this the viewport to preview a sprite in the box at the bottom of the inspector or is it within game? 

** Steps to Reproduce**
Importing this attached aesprite image will break the code described above.
[SpriteColoImportError.zip]()

**Desktop (please complete the following information):**
 - OS: Windows 10

**Comment by haywirephoenix** [comment]
I fixed this by removing the borders from my Aseprite canvas. They got automatically added when I flipped a large selection.

**Comment by martinhodler** [comment]
There is still an error when layers have data out of the range of the image. Fix to that is coming.

**Comment by martinhodler** [comment]
fixed with the commit 37a057dc0a3790556bab08a4293c62992a3ccec0
